libs/db/boltdb_test.go

```diff
@@ -0,0 +1,35 @@
+package db
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	cmn "github.com/tendermint/tendermint/libs/common"
+)
+
+func TestBoltDBNewBoltDB(t *testing.T) {
+	name := fmt.Sprintf("test_%x", cmn.RandStr(12))
+	dir := os.TempDir()
+	defer cleanupDBDir(dir, name)
+
+	db, err := NewBoltDB(name, dir)
+	require.Nil(t, err)
+	db.Close()
+}
+
+func BenchmarkBoltDBRandomReadsWrites(b *testing.B) {
```

Comment:
BTW, my mbp:
```
BenchmarkBoltDBRandomReadsWrites-12    	     100	  13983447 ns/op
BenchmarkGoLevelDBRandomReadsWrites-12    	  100000	     13278 ns/op
```
Isn't this supposed to be faster?

Reply:
LOL

Reply:
1) BenchmarkBoltDBRandomReadsWrites does not reflect our load
2) I'll double check. If bolt is slower for our load, I'll be 😢 because I spent a good portion of the day working on top of https://github.com/tendermint/tendermint/pull/3604

Reply:
3. It's not fair to compare goleveldb and bolt in this case (BenchmarkBoltDBRandomReadsWrites) because Set is synchronous in bolt! We can turn sync in this benchmark by using NoSync option

Reply:
Sounds great.   By the way, the bolt is better than leveldb in random read because it's based on B+TREE. 
  I will find some authority benchmark reports for you tomorrow. Now it's too late in China I will go for sleep.
HAPPY  MAY DAY  😁😁